### ISSUE TITLE
Exception thrown when having minimal Api Endpoints with Authorization

### ISSUE THREAD

**Issue by d4rkm0nkey** [FastEndpoints/FastEndpoints#613]
Program.cs
```csharp
var builder = WebApplication.CreateBuilder(args);
var services = builder.Services;

services
    .AddFastEndpoints()
    .AddAuthorization()
    .AddSwaggerDocument();

services
            .AddAuthorizationBuilder()
            .AddPolicy("MyAuth", policy => policy.RequireAssertion(_ => true));

var app = builder.Build();

app
    .UseCors("relaxdays")
    .UseHttpsRedirection()
    .UseRouting()
    .UseCookiePolicy()
    .UseAuthentication()
    .UseAuthorization()
    .UseFastEndpoints();

app.MapGet($"myEndpoint", () => "Hello World")
            .RequireAuthorization("MyAuth");
app.Run();
```

I get this exception when swagger is loaded:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at FastEndpoints.Swagger.OperationSecurityProcessor.Process(OperationProcessorContext context)
   at NSwag.Generation.AspNetCore.AspNetCoreOpenApiDocumentGenerator.RunOperationProcessors(OpenApiDocument document, ApiDescription apiDescription, Type controllerType, MethodInfo methodInfo, OpenApiOperationDescription operationDescription, List`1 allOperations, OpenApiDocumentGenerator generator, OpenApiSchemaResolver schemaResolver)
   at NSwag.Generation.AspNetCore.AspNetCoreOpenApiDocumentGenerator.AddOperationDescriptionsToDocument(OpenApiDocument document, Type controllerType, List`1 operations, OpenApiDocumentGenerator swaggerGenerator, OpenApiSchemaResolver schemaResolver)
   at NSwag.Generation.AspNetCore.AspNetCoreOpenApiDocumentGenerator.GenerateApiGroups(OpenApiDocumentGenerator generator, OpenApiDocument document, IGrouping`2[] apiGroups, OpenApiSchemaResolver schemaResolver)
   at NSwag.Generation.AspNetCore.AspNetCoreOpenApiDocumentGenerator.GenerateAsync(ApiDescriptionGroupCollection apiDescriptionGroups)
   at NSwag.AspNetCore.Middlewares.OpenApiDocumentMiddleware.GenerateDocumentAsync(HttpContext context)
   at NSwag.AspNetCore.Middlewares.OpenApiDocumentMiddleware.GetDocumentAsync(HttpContext context)
   at NSwag.AspNetCore.Middlewares.OpenApiDocumentMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
```

The Problem seems to be that the `Process` method in `OperationSecurityProcessor` is allowing controller endpoints, but not minimal api endpoints. The exception is thrown, because `context.ControllerType` is null.
```csharp
var epDef = epMeta.OfType<EndpointDefinition>().SingleOrDefault();

if (epDef == null)
{
    if (epMeta.OfType<ControllerAttribute>().Any()) // it is an ApiController
        return true;                                // todo: return false if the documentation of such ApiControllers is not wanted.

    throw new InvalidOperationException(
        $"Endpoint `{context.ControllerType.FullName}` is missing an endpoint description. " +
        "This may indicate an MvcController. Consider adding `[ApiExplorerSettings(IgnoreApi = true)]`");
}
   ```
   The fix could be to add this:
   ```csharp
   if(epMeta.OfType<ApiConventionNameAttribute>.Any()) // it is an Minimal Api endpoint
       return true;
   ```

**Comment by d4rkm0nkey** [comment]
But FastEndpoints probably have the `ApiConventionNameAttribute` as well, so probably check for something else that a FastEndpoint has and minimal apis don't have.

**Comment by d4rkm0nkey** [comment]
Or maybe just:
```csharp
if(context.ControllerType is null) // it is an Minimal Api endpoint
    return true;
```

**Comment by dj-nitehawk** [comment]
thanks for reporting.
should be fixed in `v5.22.0.10-beta`
